website/docs/tutorials/07-testing/02-cheatcodes/reflect.md

```diff
@@ -0,0 +1,65 @@
+# `reflect`
+```python
+def reflect(self, ids: VmConsts) -> Reflector:
+Reflector.get() -> Union[CairoStruct, RelocatableValue, int]
+def CairoStruct(typename: str) -> Type # inherits from base CairoStruct
+```
+Lazily converts Cairo object into Python custom named `CairoStruct` (complex structure) or keeps it a simple type `RelocatableValue` (pointer) or `int` (felt). It can be used to easily print and compare complex structures.
```

Comment:
Loads specified Cairo object into Python type. get() can return (describe types...)

Also prefer active over passive in docs:
>You can use it to print cairo data and compare complex structures.